### PR TITLE
Corrected bug issue template and create pull request template.

### LINKED ISSUE #154: Update Repo Templates
## TODO
- [ ] Create Pull Request Template
- [ ] Modify Issue Template

### PR DESCRIPTION
**Pull request by davidkuhta**
# Description

Corrects bug issue-template and creates pull request template in appropriate GH directory

Fixes #154 

## Type of change
- [X] Project Management fix (non-breaking change which fixes an issue)

# Checklist:

- [X] I have performed a self-review of my code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 0.0.11 (Default)
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:33]: Oooh, I think this should be 0.1.1

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:23]: I think this should be "saw" instead of "see" :)

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:10]: Good thinking, but probably not necessary, we'll see their GitHub username!

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    id: hotham-component
+    attributes:
+      label: In which Hotham component are you seeing the problem on?
+      multiple: true
+      options:
+        - Core
+        - Example
+        - Simulator
+        - Debug 
+  - type: dropdown
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:47]: An "Other" option here would be good!

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        - Linux
+        - MacOS
+        - Windows
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:70]: Nice one

File: .github/PULL_REQUEST_TEMPLATE/pull-request-task.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Checklist:
+
+- [ ] I have performed a self-review of my code
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (kanerogers) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-task.md:22]: This is awesome! I would also add:

> - [ ] I have added tests for my code to help prevent regressions in future

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 0.0.11 (Default)
+        - latest (Edge)
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:34]: I'd probably just leave this as "latest".

**Review by kanerogers — DISMISSED** [review]
Fantastic thinking @davidkuhta! 

I've just added a couple of quick notes; other than that this is looking good to go!

File: .github/ISSUE_TEMPLATE/bug-task.yaml (hunk 3 above)
Comment (davidkuhta) [reply .github/ISSUE_TEMPLATE/bug-task.yaml:10]: Making a note here, that at this point I would agree, but in the future depending on the project size and org, sometimes the quickest contact method isn't via GH.

File: .github/ISSUE_TEMPLATE/bug-task.yaml (hunk 2 above)
Comment (davidkuhta) [reply .github/ISSUE_TEMPLATE/bug-task.yaml:23]: Was really tempted to add a dropdown for real world vs VR world.

File: .github/PULL_REQUEST_TEMPLATE/pull-request-task.md (hunk 6 above)
Comment (kanerogers) [reply .github/PULL_REQUEST_TEMPLATE/pull-request-task.md:23]: Sorry to nitpick this but I think just a single line about tests is sufficient!

File: .github/ISSUE_TEMPLATE/bug-task.yaml [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+        - Debug 
+        - Other
+  - type: dropdown
+    id: vr-system
+    attributes:
+      label: What VR System are you seeing the problem on?
+      multiple: false
+      options:
+        - Quest-2
+        - Quest-1
```
Comment (kanerogers) [inline .github/ISSUE_TEMPLATE/bug-task.yaml:47]: @davidkuhta Whoops! I think I meant a dropdown for VR system!

File: .github/ISSUE_TEMPLATE/bug-task.yaml (hunk 2 above)
Comment (kanerogers) [reply .github/ISSUE_TEMPLATE/bug-task.yaml:15]: Hahaha, yes indeed. :)

PS. this should be updated to saw (consistency of tenses between "what happened" and "what you saw")

**Review by kanerogers — CHANGES_REQUESTED** [review]
@davidkuhta Almost there! Just a couple of tiny nits left

File: .github/ISSUE_TEMPLATE/bug-task.yaml (hunk 8 above)
Comment (davidkuhta) [reply .github/ISSUE_TEMPLATE/bug-task.yaml:47]: The VR selection is a dropdown?

**Review by kanerogers — APPROVED** [review]
Perfect! 👨🏻‍🍳

**Pull request merged**